File: Ravi-Teja/ReverseBits.cpp
class Solution {
public:
    uint32_t reverseBits(uint32_t n) {
        uint32_t a=(1<<31), b=1;
        while(a>b){
            if(((n&a)!=a && (n&b)==b) || ((n&a)==a && (n&b)!=b)){
                n=(n^a);
                n=(n^b);
            }
            a=a>>1;
            b=b<<1;
        }
        return n;
    }
};

File: Parth Mathur/reverseBits.java
public class Solution {
    public int reverseBits(int n) {
        int res = 0;
        for(int i=0;i<32;i++){
            int bit = (n >> i) & 1;
            res = res + (bit<<(31-i));
        }
        return res;
    }
}
